### DISCUSSION TITLE
Am I using cmd wrong?

### DISCUSSION THREAD

**Discussion by axkirillov** [ibhagwan/fzf-lua#1311] [Q&A] [answered]
here is a function that I wrote
```lua
local branch_diff = function()
	local default_branch = vim.fn.system("git rev-parse --symbolic-full-name refs/remotes/origin/HEAD | sed 's!.*/!!'") or
	"main"
	local command = "git diff --name-only $(git merge-base HEAD " .. default_branch .. " )"
	require 'fzf-lua'.files({ cmd = command })
end

vim.keymap.set("n", "<leader>pf", branch_diff, { noremap = true, silent = true })

```

Command itself works without problems in the shell. But in fzf lua I see these entries

![image](https://github.com/ibhagwan/fzf-lua/assets/32141102/83047a61-4fb7-4e36-89c9-efb8310f5aed)

**Comment by ibhagwan** [comment, marked as answer]
> Command itself works without problems in the shell. But in fzf lua I see these entries

Fzf-lua can’t make up output on its own, add `debug=true` to see what command you’re sending to fzf-lua, your issue is there most likely.

**Reply by axkirillov** [reply]
you were right, thank you, the issue was entirely with the command itself 👍🏻
